frontend/src/components/VSearchResultsGrid/VImageCell.vue

```diff
@@ -20,8 +20,15 @@
           ref="img"
           loading="lazy"
           class="block w-full rounded-sm object-cover"
-          :class="isSquare ? 'h-full' : 'margin-auto'"
-          :alt="image.title"
+          :class="[
+            isSquare ? 'h-full' : 'margin-auto',
+            { 'scale-105 blur': shouldBlur },
```

Comment:
I think we should use either `blur-xl` or `blur-lg`. Currently, some images are too clear and don't really hide the content.

Reply:
I picked a lower level of blur to help the user get a general idea of the shape of the contents of the image because with more blur, only some colour clues are left. 

But I agree with your reasoning and will increase the amount.

Reply:
Is there a possibility of the genre containing anything sensitive? I don't think that is possible.

Reply:
If anything, genre will help people make a determination about whether or not they want to look at an audio file. I recommend we leave it unburred